nesta/packages/unsdg_nlp_utils/bigram_gensim_creation.py

```diff
@@ -0,0 +1,9 @@
+from gensim.models.phrases import Phrases, Phraser
+
+def generate_bigrams(ls, bigram_model):

```

Comment:
I don't think this function is needed, as it performs a function that you could do using the two arguments anyway.